vendor/bat-native-confirmations/src/bat/confirmations/internal/payments.cc

```diff
@@ -65,12 +65,13 @@ bool Payments::SetFromDictionary(base::DictionaryValue* dictionary) {
     PaymentInfo payment;
 
     // Balance
-    auto* balance_value = dictionary->FindKey("balance");
-    if (!balance_value || !balance_value->is_double()) {
+    const base::Optional<double> balance_value =
```

Comment:
Unit tests highlighted that `0` in the `confirmations.json` state file for the `balance` would return `false` due to `is_double()` failing